common-libs/vc-common/src/vc/v1.ts

```diff
@@ -11,7 +11,7 @@ export type MaybeArray<T> = T | Array<T>
 export type VCV1Subject<Data extends SimpleThing> = {
   '@context'?: string
   id?: string
-  data: Data
+  data?: Data
```

Comment:
Shouldn't it be
```
export type VCV1Subject<Data extends SimpleThing> = {
  '@context'?: string
  id?: string
} & ({ data: Data } | Data)
```
instead? Updated tests make it seem so.

Reply:
Indeed, this is better. Though, since vc-data is deprecated, I thought these types are no longer needed, but I might be wrong and they might still be useful.